requirements/docs-lint.txt

```diff
@@ -0,0 +1,2 @@
+-r pip.txt
```

Comment:
Not sure if we need all the packages from pip.txt, but just to follow similar convention, and we need Sphinx installed.

Reply:
No we don't need them. That's the idea of using a different file.

Add here just the packages that are needed. As far as I know, `sphinx` and `rstcheck` (also, do not pin them)

Reply:
No we don't need them. That's the idea of using a different file.

Add here just the packages that are needed. As far as I know, `sphinx` and `rstcheck` (also, do not pin them)